deploy/docker/Dockerfile-slim

```diff
@@ -11,21 +11,34 @@ RUN make build_with_edgex
 
 FROM debian:10
 
+ENV MAINTAINER="emqx.io" \
+    KUIPER_HOME="/kuiper" \
+    KUIPER__BASIC__CONSOLELOG=true
+
+ARG KUIPER_USER="kuiper"
+ARG KUIPER_USER_ID="1001"
+
 COPY ./deploy/docker/docker-entrypoint.sh /usr/bin/docker-entrypoint.sh
 COPY --from=builder /go/kuiper/kuiper_conf_util /usr/bin/kuiper_conf_util
 COPY --from=builder /go/kuiper/_build/kuiper-* /kuiper/
 
-RUN apt update && apt install -y pkg-config libczmq-dev wget
-
-WORKDIR /kuiper
+RUN apt update && apt install -y pkg-config libczmq-dev wget && \
+    adduser -q --no-create-home --uid ${KUIPER_USER_ID} --disabled-password --shell /usr/sbin/nologin --gecos "" ${KUIPER_USER}
 
-ENV MAINTAINER="emqx.io"
-ENV KUIPER_HOME /kuiper
-ENV KUIPER__BASIC__CONSOLELOG true
+WORKDIR ${KUIPER_HOME}
 
-VOLUME ["/kuiper/etc", "/kuiper/data", "/kuiper/plugins", "/kuiper/log"]
+VOLUME ["${KUIPER_HOME}/etc", "${KUIPER_HOME}/data", "${KUIPER_HOME}/plugins", "${KUIPER_HOME}/log"]
 EXPOSE 9081 20498
 
+# Set appropriate ownership to allow binary full access to KUIPER_HOME dir
+RUN chown -Rh ${KUIPER_USER}:${KUIPER_USER} ${KUIPER_HOME}
+
+# Create taos db directory and set kuiper to owner
+RUN mkdir -p /usr/local/taos && chown -Rh ${KUIPER_USER}:${KUIPER_USER} /usr/local/taos
+
```

Comment:
I think `adduser` and `chown` can be combined in the same `RUN`, which makes it easier to read, don't you think?

Reply:
No, they are separated because of build timing - "root" has to create the user and perform build processes, and then assign ownership after the build processes are complete.

Reply:
Updated - push coming shortly.